internal/extensions/extensions.go

```diff
@@ -8,6 +8,9 @@ type extInfo struct {
 }
 
 var ExtMap = map[types.Path]extInfo{
+	"hasTag": {Args: 2, IsMethod: true},
+	"getTag": {Args: 2, IsMethod: true},
+
```

Comment:
As I mentioned above, these methods should be treated as  first-class operators, not extension methods.

Reply:
Thank you, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/4f433e34c428f2de144c83035ba1fd0a313a904c